schunk_lwa4d/package.xml

```diff
@@ -18,25 +18,26 @@
   <build_depend>roslaunch</build_depend>
   <build_depend>rostest</build_depend>
 
+  <run_depend>xacro</run_depend>
+  <run_depend>rostopic</run_depend>
+  <run_depend>topic_tools</run_depend>
+  <run_depend>cob_description</run_depend>
```

Comment:
Yes, because the launch test requires it:
https://github.com/ipa320/schunk_robots/blob/indigo_dev/schunk_lwa4d/urdf/test/urdf.test#L4